.github/workflows/ovn-kubernetes.yml

```diff
@@ -0,0 +1,169 @@
+name: ovn-kubernetes
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+  schedule:
+  # Run Sunday at midnight
+  - cron: '0 0 * * 0'
+
+env:
+  GO_VERSION: "1.16.3"
+  K8S_VERSION: v1.20.2
+  KIND_CLUSTER_NAME: ovn
+  KIND_INSTALL_INGRESS: true
+  KIND_ALLOW_SYSTEM_WRITES: true
+  # This skips tests tagged as Serial
+  # Current Serial tests are not relevant for OVN
+  PARALLEL: true
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-20.04
```

Comment:
Any specific reason why this task runs on ubuntu-20.04 while the e2e one runs on ubuntu-latest?  Checking upstream ovn-kubernetes repo I see all tasks run on ubuntu-latest.

Reply:
ubuntu-latest is moving, not so often, right now is ubuntu-20.04, 
but this way you don't have to go to the same I had to go when jobs started failing because the version changed without any warning :)

Reply:
Sure, but should we make it ubuntu-latest everywhere then?

Reply:
if there is one change , it has to be `ubuntu-20.04` everywhere, since the switch of ubuntu-latest to another version will break current jobs without notification

Reply:
Ah, right. :)